arch/arm/boot/dts/broadcom/bcm2712-rpi.dtsi

```diff
@@ -45,6 +45,34 @@
 		trickle-charge-microvolt = <0>;
 	};
 
+	nvmem_otp: nvmem_otp {
+		compatible = "raspberrypi,rpi-otp";
+		firmware = <&firmware>;
+		reg = <0 192 0>;
+		status = "okay";
+	};
+
+	nvmem_customer: nvmem_customer {
+		compatible = "raspberrypi,rpi-otp";
+		firmware = <&firmware>;
+		reg = <1 8 1>;
+		status = "okay";
+	};
+
+	nvmem_mac: nvmem_mac {
```

Comment:
Is it possible for a user to make the MAC addresses visible but read-only via device-tree or module parameters?